src/defaultPhrases.js

```diff
@@ -0,0 +1,176 @@
+const closeDatePicker = 'Close';
+const focusStartDate = 'Focus on start date';
+const clearDate = 'Clear Date';
+const clearDates = 'Clear Dates';
+const jumpToPrevMonth = 'Jump to previous month';
+const jumpToNextMonth = 'Jump to next month';
+const keyboardShortcuts = 'Keyboard Shortcuts';
+const showKeyboardShortcutsPanel = 'Show the keyboard shortcuts panel';
+const hideKeyboardShortcutsPanel = 'Hide the keyboard shortcuts panel';
+const toggleKeyboardShortcutsPanel = 'Toggle the keyboard shortcuts panel';
+const enterKey = 'Enter key';
+const leftArrowRightArrow = 'Left Arrow/Right Arrow';
+const upArrowDownArrow = 'Up Arrow/Down Arrow';
+const pageUpPageDown = 'Page Up/Page Down';
+const homeEnd = 'Home/End';
+const escape = 'Escape';
+const questionMark = 'Question mark';
+const selectFocusedDate = 'Select the currently focused date';
+const moveFocusByOneDay = 'Decrement/Increment currently focused day by 1 day';
```

Comment:
I feel like we can nit the text, but it would probs make sense until someone can look at the copy and give us recs.